.agents/skills/cuopt-server-api-python/assets/milp_basic/client.py

```
@@ -0,0 +1,82 @@
+            "upper_bounds": ["inf", "inf"],
+            "lower_bounds": [0.0, 0.0],
+        },
+        "variable_types": ["integer", "continuous"],
+        "maximize": True,
+        "solver_config": {
+            "time_limit": 120,
+            "tolerances": {"mip_relative_gap": 0.01},
+        },
+    }
+
+    response = requests.post(
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)